.github/workflows/test.yml

```diff
@@ -17,23 +17,25 @@ jobs:
     strategy:
       matrix:
         node-version:
-          - '20.9'
-          - '18.17'
-          - '16.20'
+          - '22.11'
+          - '20.18'
+          - '18.20'
         os:
```

Comment:
Can downgrade to eslint@8 if we want to keep Node.js 16 CI but that version is already EOL'd.

Reply:
Seems reasonable to drop it, and if we migrate this repo to Vitest it needs 18+ anyway.